lms/djangoapps/branding/api.py

```diff
@@ -178,9 +178,9 @@ def _footer_navigation_links():
             ("about", marketing_link("ABOUT"), _("About")),
             ("blog", marketing_link("BLOG"), _("Blog")),
             ("news", marketing_link("NEWS"), _("News")),
-            ("faq", marketing_link("FAQ"), _("FAQs")),
+            ("help-center", "https://support.edx.org", _("Help Center")),
```

Comment:
Why is `https://support.edx.org` being harcoded here instead of being pulled from configuration?